man/udevadm.xml

```diff
@@ -363,7 +363,9 @@
           <term><option>-e</option></term>
           <term><option>--exit</option></term>
           <listitem>
-            <para>Signal and wait for systemd-udevd to exit.</para>
+            <para>Signal and wait for systemd-udevd to exit. Note that systemd-udevd.service contains
```

Comment:
the unit name should be enclosed in `<filename></filename>` please. Here and below. it's what we usually do elsewhere too.